packages/stores/types.ts

```diff
@@ -335,7 +335,7 @@ type TPortfolioPosition = {
 type TPortfolioStore = {
     active_positions: TPortfolioPosition[];
     all_positions: TPortfolioPosition[];
-    error: string;
+    error: TCommonStoreError;
```

Comment:
@henry-deriv please revert this change, explanation is here https://github.com/kate-deriv/deriv-app/pull/11/files#r1252641368

Reply:
reverted